### ISSUE TITLE
UWP Sqlite 'VACUUM; error

### ISSUE THREAD

**Issue by Fernand-Delavy** [dotnet/efcore#20253]
[Test.zip](https://github.com/dotnet/efcore/files/4318467/Test.zip)
Sorry for my bad English.
Very small App VS 2019 Test attached to see the problem, rebuid it and run.
If you run as it work. if you update Microsoft.Data.Sqlite to version 3.1.2 or > 2.2.6 it crash.
Id did talk the problem to Microsoft, response
 
Svet Bonev [MSFT] 
Hello! Thank you for your feedback!
We have determined that the feedback applies to another product: Entity Framework Core. We recommend that you submit the feedback to that product team at https://github.com/aspnet/EntityFrameworkCore. Thank you and we value your contribution to Developer Community.

Thanks in advance

**Comment by bricelam** [comment]
Works outside of UWP.
Works using winsqlite3.dll. (the version of SQLite that ships as part of the Windows SDK)

``` diff
- <PackageReference Include="Microsoft.Data.Sqlite">
+ <PackageReference Include="Microsoft.Data.Sqlite.Core">
    <Version>3.1.2</Version>
  </PackageReference>
+ <PackageReference Include="SQLitePCLRaw.bundle_winsqlite3">
+   <Version>2.0.2</Version>
+ </PackageReference>
```

**Comment by ajcvickers** [comment]
Closing as external since this is now tracked by https://github.com/ericsink/SQLitePCL.raw/issues/339

**Comment by Fernand-Delavy** [comment]
Hi Sir,

I see you did close the tiquet without a solution for UWP ! Not very fair.
The problem is on the version 5.0.0 preview 2.20120.8 too.
Regards.


Fernand Delavy
Rue Carlo Hemmerling 11
CH-1804 Corsier-sur-Vevey
Email :     fernand.delavy@outlook.com
Tél :          +41 21 922 85 65
Mobile :  +41 79 668 29 94

De : Brice Lambson <notifications@github.com>
Envoyé : jeudi, 12 mars 2020 23:30
À : dotnet/efcore <efcore@noreply.github.com>
Cc : Fernand-Delavy <fernand.delavy@outlook.com>; Author <author@noreply.github.com>
Objet : Re: [dotnet/efcore] UWP Sqlite 'VACUUM; error (#20253)


Works outside of UWP.
Works using winsqlite3.dll. (the version of SQLite that ships as part of the Windows SDK)

- <PackageReference Include="Microsoft.Data.Sqlite">

+ <PackageReference Include="Microsoft.Data.Sqlite.Core">

    <Version>3.1.2</Version>

  </PackageReference>

+ <PackageReference Include="SQLitePCLRaw.bundle_winsqlite3">

+   <Version>2.0.2</Version>

+ </PackageReference>

**Comment by ajcvickers** [comment]
@Fernand-Delavy It's closed here because the issue is external to EF. The linked issue https://github.com/ericsink/SQLitePCL.raw/issues/339 is still open and I see you are being helped there. We have a lot of issues to track so we try not to track the same thing in multiple places.

**Comment by bricelam** [comment]
Sounds like this may be fixed by PR #20077
